src/sync/mod.rs

```diff
@@ -79,7 +80,7 @@ pub async fn begin_sync(
             client_id: client_id.clone(),
             mutations: push_mutations,
         };
-        logger.debug(str!("Starting push..."));
+        debug!("", "Starting push...");
```

Comment:
here and below we should be using the lc not the empty string (maybe search and replace gone awry?)

Reply:
eep, thanks.